Rocket.Chat/Controllers/Settings/SettingsViewModel.swift

```diff
@@ -73,7 +85,7 @@ final class SettingsViewModel {
 
     internal func numberOfRowsInSection(_ section: Int) -> Int {
         switch section {
-        case 0: return 3
+        case 0: return (canChangeAppIcon ? 4 : 3)
```

Comment:
Looks good now, but we'll have to change this logic in the future, to avoid having to deal with this type of code... 👍

Reply:
Yup, my next goal is to refactor Settings's controller